test/jl/tests.jl

```diff
@@ -132,4 +132,18 @@ end
 
 test5()
 
+################################################################################################
+#
+# test 6: Conversion of images from Images.jl to OpenCV Mat 
+#
+################################################################################################
+
+function test6()
+   println("test 6: Conversion of images from Images.jl to OpenCV Mat")
+   filename = joinpath(Pkg.dir("OpenCV"), "./test/images/mandrill.jpg")
+   image = imread(filename)
+   dst = Mat(int(rows(image)), int(cols(image)), CV_8UC1)
```

Comment:
I believe`int` is deprecated in `0.6.0`, so we to use `Int(round(rows(image))` instead.  Please check this and confirm that test5 and test6 pass on your system.